src/parachain/nomination.ts

```diff
@@ -102,12 +102,24 @@ export interface NominationAPI extends TransactionAPI {
      *
      * @param nominatorId Id of user who nominated to one or more vaults
      * @param currency The currency of the nominations
-     * @returns The rewards (in InterBTC a nominator has accumulated)
+     * @returns The rewards a currently active nominator has accumulated
      */
-    getNominatorRewards<C extends CurrencyUnit>(
+    getActiveNominatorRewards<C extends CurrencyUnit>(
```

Comment:
Minor nit, old naming was fine - Nominators are never not active.

Reply:
They can be not active if they don't have active nomination but still have rewards to collect. Maybe another name would describe the situation better, but we need to make the distinction

Reply:
What are we making the distinction on? Is there a `getUnactiveNominatorRewards`?

Reply:
Yes, but the naming (perhaps not the best) is `getNominatorReward` https://github.com/interlay/interbtc-js/pull/315/files#diff-43eba5e3e371acf89540a6f85d6a52f42e005c8609363a860b60be839c69d2c6R118

Reply:
Ah I missed that, okay then makes sense to me!